rtmbot.py

```diff
@@ -145,14 +147,42 @@ def main_loop():
 
 def parse_args():
     parser = ArgumentParser()
-    parser.add_argument(
+
+    group = parser.add_mutually_exclusive_group()
+    group.add_argument(
         '-c',
         '--config',
         help='Full path to config file.',
         metavar='path'
     )
+    group.add_argument(
+        '-s',
+        '--settings',
+        help='Python path to settings module.',
+        metavar='module'
+    )
+
+    # Allows the bot to be run with zero arguments and use a Python module
+    # instead of a YAML file.
+    if os.getenv('TEAMBOT_SETTINGS_MODULE') is not None:
+        parser.set_defaults(settings=os.getenv('TEAMBOT_SETTINGS_MODULE'))
+    else:
+        parser.set_defaults(config='rtmbot.conf')
+
     return parser.parse_args()
 
+def get_config(args):
+    if args.config:
+        config = yaml.load(file(args.config, 'r'))
+    elif args.settings:
+        s = importlib.import_module(args.settings)
+        get_s = functools.partial(getattr, s)
+        config = {
```

Comment:
@tdeck I'm not super happy with this, i.e. explicitly listing the configs here.

Maybe something like `defaultSettings.update(userSettings)` would be nicer, but I wanted to keep it simple.